avatar-screenshotter.js

```diff
@@ -84,10 +84,40 @@ export const screenshotPlayer = async ({
       _animate(now, timeDiff);
     };
   };
+  const localVector = new THREE.Vector3();
+  const boundingBox = new THREE.Box3();
   const _updateTarget = (timestamp, timeDiff) => {
+    const neckPosition = localVector.setFromMatrixPosition(player.avatar.modelBones.Head.savedMatrixWorld);
+    let avatarHighestPos = 0;
+    let tempMesh = null;
+    player.avatar.model.traverse(o => {
+      if (o.isMesh) {
+        if(!o.geometry.boundingBox){
+          const position = o.geometry.attributes.position;
+          boundingBox.setFromBufferAttribute( position );

```

Comment:
This is still slow.

Also, this is available as a method on the `Geometry`: https://threejs.org/docs/#api/en/core/BufferGeometry.boundingBox

Reply:
Yes, but there are a few avatars do not have `boundingBox`, so need to compute the bounding box manually. 
This line is for the avatars that do not have the bounding box

Reply:
But you can call computeBoundingBox instead.

Reply:
Hi @avaer, but I found that computeBoundingBox is not that accurate.
![image](https://user-images.githubusercontent.com/60634884/184451991-f0f3318e-634c-4d80-b05c-e635b9971174.png)

In this picture, the yellow box is the result by using `o.geometry.computeBoundingBox();` and the red ball is the result by using the method above. 

After I tested it, I found it is caused by `morphAttributesPosition` in three.js.
![unknown (1)](https://user-images.githubusercontent.com/60634884/184452668-21b6618a-7704-4be5-82d3-231594b91270.png)